conf/alembic.ini

```diff
@@ -0,0 +1,37 @@
+[alembic]
+script_location = lti:migrations
+sqlalchemy.url: postgresql://postgres@localhost:5433/postgres
```

Comment:
Apart from editing the `sqlalchemy.url` setting by the way, and deleting some comments, this `alembic.ini` file is as generated by `alembic init`